releasenotes/notes/1.3/add-mcx-plugins-85e5b248692a36db.yaml

```diff
@@ -17,7 +17,7 @@ features_transpiler:
     * :class:`.MCXSynthesisNoAuxV24`, based on :func:`.synth_mcx_noaux_v24`.
     * :class:`.MCXSynthesisGrayCode`, based on :func:`.synth_mcx_gray_code`.
 
-    As well:
+    As well as:
 
     * :class:`.MCXSynthesisDefault`, choosing the most efficient synthesis
       method based on the number of clean and dirty ancilla qubits available.
```

Comment:
I can't comment there but on the line below there's a typo:

"As an example, consider ~~how~~ the transpilation of the following circuit"